pipelines/matrix/conf/base/run_comparison/parameters.yml

```diff
@@ -0,0 +1,69 @@
+run_comparison:
```

Comment:
This file looks a bit messy, can we clean this up a bit better so its more consumable